optimus_prime/Source.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+
+
+bool is_prime(int number) {
+	if (number < 2) {
```

Comment:
This is an "ok"-ish way to deal with negative numbers, but why in runtime? 
Receive an unsigned integer and then enforce the number's positivity in compile time!